examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,125 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are stored in previous post processing pulled from an H5 file. The inverse of the T1(p) data is taken to produce the R1(p) data which is fit using the polyfit of the inverse of krho with two degrees of freedom. The R1(p) fit is then fed into the fitting routine for the enhancement data prior to normalization. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex
+from scipy.io import loadmat
+
+h5_file = "ras.h5"
+nodename = "230706_M67_a"
+
+
+# {{{R1p expression
+def R1p_expression(
+    p,
+):  # symbolic expression for R1p used in both fitting the R1p data as well as the symbolic expression for E(p)
+    expression = (T10_p[0] + T10_p[1] * p) ** -1 + (
+        Ep.get_prop("acq_params")["concentration"]
+        / (krho_inv_coeff[0] + krho_inv_coeff[1] * p)
+    )
+    return expression
+
+
+# }}}
```

Comment:
you should not have a function to define the expression.   You should just define the expression inline